package.json

```diff
@@ -50,6 +50,7 @@
     "ajv-formats": "^2.1.1",
     "axios": "^1.6.7",
     "fastify-plugin": "^4.5.0",
+    "lodash": "^4.17.21",
```

Comment:
I'd not add a new dependency to the library. We can achieve the same behavior with a bunch of if statements